src/main/java/com/hackclub/hccore/slack/SlackBot.java

```diff
@@ -74,6 +73,10 @@ public class SlackBot implements Listener {
   public static final String playerServerLeaveAvatarUrl = "https://cloud-if9tepzbn-hack-club-bot.vercel.app/0hccoreleave.png";
   public static final String playerServerJoinAvatarUrl = "https://cloud-if9tepzbn-hack-club-bot.vercel.app/1hccorejoin.png";
   public static final String playerAdvancementAvatarUrl = "https://cloud-obk2f29h4-hack-club-bot.vercel.app/0achievement.png";
+  private final HCCorePlugin plugin;
+  private final SocketModeApp socket;
+  private final String commandBase;
+  private final HashMap<String, String> mcUuidVerificationCodes = new HashMap<>();
```

Comment:
should this be changed for expiring codes?
easiest way is probably apache commons collections4 PassiveExpiringMap
 https://commons.apache.org/proper/commons-collections/apidocs/org/apache/commons/collections4/map/PassiveExpiringMap.html

Reply:
yeah that makes sense, how long do you think each code should live?

Reply:
10.mins.
Also make sure to say this in the join message

Reply:
Actually, you could make it configurable? Not required